src/uu/tail/src/tail.rs

```diff
@@ -26,12 +26,13 @@ use args::{FilterMode, Settings, Signum, parse_args};
 use chunks::ReverseChunks;
 use follow::Observer;
 use memchr::{memchr_iter, memrchr_iter};
-use paths::{FileExtTail, HeaderPrinter, Input, InputKind, MetadataExtTail};
+use paths::{FileExtTail, HeaderPrinter, Input, InputKind};
 use same_file::Handle;
 use std::cmp::Ordering;
 use std::collections::HashMap;
 use std::fs::File;
 use std::io::{self, BufReader, BufWriter, ErrorKind, Read, Seek, SeekFrom, Write, stdin, stdout};
+use std::os::unix::fs::MetadataExt;
```

Comment:
You have to use a `#[cfg]` attribute as this line doesn't work on Windows.
```suggestion
#[cfg(unix)]
use std::os::unix::fs::MetadataExt;
```